### PR TITLE
Featuers branch

### PR DESCRIPTION
**Pull request by mbdesigns1989**
**Features branch**

1. Create a table for storing messages and create 5 different greetings.

2. Create an API endpoint that selects a random greeting from the table

3. Create a static view that will be the root of the app.

4. Create the App component with react-router and render it in the static view.

5. Create the Greeting component that will display a greeting.

6. Create a store, an action, and a reducer that will connect to the API endpoint to get the random greeting.

7. Display the random greeting in your Greeting component.

### CODE REVIEW

File: app/javascript/redux/getText.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    if (data.text) {
+      return {
+        success: true,
+        data:data.text,
+      };
+    }
+  } catch (err) {
+    return { success: false, err: err.response.data.text };
+  }
+});
```
Comment (Gambit142) [inline app/javascript/redux/getText.js]: - [ ] When I run your application, I get this error in the console:
 
Screenshot 2023-07-10 173433

I would advise you to fetch the Greetings data using this URL `"http://localhost:3000/api/greetings"`. 💻

File: README.md [hunk 2]
```diff
@@ -1 +1,101 @@
...
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+ 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT]
```
Comment (Gambit142) [inline README.md]: - [ ] Kindly include a valid LICENSE link to the LICENSE section in your README file. Adding an open-source license to your repository ensures that others can use, copy, modify, and contribute back to your project. It's an important step when creating an open-source project. 😎

**Review by Gambit142 — CHANGES_REQUESTED** [review]
Hi @mbdesigns1989 👋, 

Good job so far!

## Highlights
- All linter checks passed ✔️ 
- Use of `jsbundling-rails` gem ✔️ 
- Configured backend properly ✔️ 
- Use of react and redux ✔️ 

You still need to work on some issues to go to the next project, but you are almost there!

## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

You can also consider the following:

N/A

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### Please tag me in your question so I can receive the notification. You can tag me by typing @Gambit142

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: README.md (hunk 2 above)
Comment (julie-ify) [inline README.md]: - [ ] Kindly ensure you link the MIT to the file you created as the previous reviewer requested. This should be a clickable link that takes the user to the MIT file.

**Review by julie-ify — CHANGES_REQUESTED** [review]
Hi @mbdesigns1989,

## Highlights!
- Displays greetings ✔️ 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (peterrobert) [reply README.md]: This change has been done well

**Review by peterrobert — APPROVED** [review]
Hi @mbdesigns1989, 


### Highlights!

[ ] Good feature branch name ✔️
[ ] Descriptive pull request ✔️


Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread or tag me @peterrobert if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**